### ISSUE TITLE
profile: sdding new group to favorites causes weird things

### ISSUE THREAD

**Issue by sethfork** [tloncorp/landscape-apps#1879]
Since adding a new favorite group to my profile in landscape, I've experienced a collection of weird things that started with a firehose of `@p is your neighbor`, and `%group-store path/to/group/join` logs - the logs matched the group in question, which is hosted by my ship. After some of those wound down, I've consistently been getting `%group-store: on-peek on path` and ames `bail` errors with increasing # of packets dropped (100 -> 200 -> 300 ... 600). @arthyn mentioned this is due to `contact-store` from old groups being used, and that you're actively working on replacing this / the use of the old federation method, which is good to hear. The bail exits / ames errors are starting to get concerning though, so I'm wondering if there's a recommended way I could address that in the mean time - and if using continuing to use [and add to contact store](https://github.com/tloncorp/landscape-apps/issues/1305) is worth the risk. Bail errors included below.

**To Reproduce:**
1. Add new, high member count group to your favorites in landscape.
2. Observe dojo

**Context:**
- Planet
- Self hosted, DO droplet

```
[%poke %crud]
bar-stack=[i=[i=//ames t=~] t=~]
call: failed
/sys/vane/ames/hoon:<[1.121 3].[1.153 20]>
/sys/vane/ames/hoon:<[1.123 3].[1.153 20]>
/sys/vane/ames/hoon:<[1.124 3].[1.153 20]>
/sys/vane/ames/hoon:<[1.126 3].[1.153 20]>
/sys/vane/ames/hoon:<[1.127 5].[1.151 7]>
/sys/vane/ames/hoon:<[1.130 5].[1.151 7]>
/sys/vane/ames/hoon:<[1.131 7].[1.134 9]>
/sys/vane/ames/hoon:<[1.133 16].[1.133 60]>
/sys/vane/ames/hoon:<[1.615 5].[1.615 43]>
/sys/vane/ames/hoon:<[1.621 5].[1.641 17]>
/sys/vane/ames/hoon:<[1.622 5].[1.641 17]>
/sys/vane/ames/hoon:<[1.624 5].[1.641 17]>
/sys/vane/ames/hoon:<[1.626 5].[1.641 17]>
/sys/vane/ames/hoon:<[1.630 5].[1.641 17]>
/sys/vane/ames/hoon:<[1.724 5].[1.776 62]>
/sys/vane/ames/hoon:<[1.725 5].[1.776 62]>
/sys/vane/ames/hoon:<[1.731 5].[1.776 62]>
/sys/vane/ames/hoon:<[1.735 5].[1.776 62]>
/sys/vane/ames/hoon:<[1.740 5].[1.776 62]>
/sys/vane/ames/hoon:<[1.741 5].[1.776 62]>
[%ames-crash-on-packet-from ~bidbel]
/sys/vane/ames/hoon:<[1.742 5].[1.776 62]>
/sys/vane/ames/hoon:<[1.743 7].[1.743 76]>
/sys/vane/ames/hoon:<[458 3].[469 63]>
/sys/vane/ames/hoon:<[459 3].[469 63]>
/sys/vane/ames/hoon:<[460 5].[460 44]>
[%ames-sndr-tick 0b1]
/sys/vane/ames/hoon:<[460 42].[460 44]>
hear: bail: %exit

[%poke %hear]
bar-stack=[i=[i=//ames t=~] t=~]
call: failed
/sys/vane/ames/hoon:<[1.121 3].[1.153 20]>
/sys/vane/ames/hoon:<[1.123 3].[1.153 20]>
/sys/vane/ames/hoon:<[1.124 3].[1.153 20]>
/sys/vane/ames/hoon:<[1.126 3].[1.153 20]>
/sys/vane/ames/hoon:<[1.127 5].[1.151 7]>
/sys/vane/ames/hoon:<[1.130 5].[1.151 7]>
/sys/vane/ames/hoon:<[1.136 5].[1.151 7]>
/sys/vane/ames/hoon:<[1.138 14].[1.138 53]>
/sys/vane/ames/hoon:<[1.615 5].[1.615 43]>
/sys/vane/ames/hoon:<[1.621 5].[1.641 17]>
/sys/vane/ames/hoon:<[1.622 5].[1.641 17]>
/sys/vane/ames/hoon:<[1.624 5].[1.641 17]>
/sys/vane/ames/hoon:<[1.626 5].[1.641 17]>
/sys/vane/ames/hoon:<[1.630 5].[1.641 17]>
/sys/vane/ames/hoon:<[1.724 5].[1.776 62]>
/sys/vane/ames/hoon:<[1.725 5].[1.776 62]>
/sys/vane/ames/hoon:<[1.731 5].[1.776 62]>
/sys/vane/ames/hoon:<[1.735 5].[1.776 62]>
/sys/vane/ames/hoon:<[1.740 5].[1.776 62]>
/sys/vane/ames/hoon:<[1.741 5].[1.776 62]>
[%ames-crash-on-packet-from ~bidbel]
/sys/vane/ames/hoon:<[1.742 5].[1.776 62]>
/sys/vane/ames/hoon:<[1.743 7].[1.743 76]>
/sys/vane/ames/hoon:<[458 3].[469 63]>
/sys/vane/ames/hoon:<[459 3].[469 63]>
/sys/vane/ames/hoon:<[460 5].[460 44]>
[%ames-sndr-tick 0b1]
/sys/vane/ames/hoon:<[460 42].[460 44]>
crud: bail: %exit
ames: packet failed (600 total)
```

**Comment by sethfork** [comment]
Additional logs I've been seeing since adding a new favorite:
```
ames: heard bad crypto (400 total), check azimuth state
```
```
ames: czar at reg.urbit.org: not found (b)
```

**Comment by sethfork** [comment]
Latest bail has 4100 dropped packets. It definitely appears to be increasing linearly, but I'm not sure if that means its cumulative or higher severity or what. There's also a few stars and ships that I regularly get this from (~bitbel ~racwet in particular). Could it be that they're running and/or sponsoring old desks?
```

[%poke %crud]
bar-stack=[i=[i=//ames t=~] t=~]
call: failed
/sys/vane/ames/hoon:<[1.121 3].[1.153 20]>
/sys/vane/ames/hoon:<[1.123 3].[1.153 20]>
/sys/vane/ames/hoon:<[1.124 3].[1.153 20]>
/sys/vane/ames/hoon:<[1.126 3].[1.153 20]>
/sys/vane/ames/hoon:<[1.127 5].[1.151 7]>
/sys/vane/ames/hoon:<[1.130 5].[1.151 7]>
/sys/vane/ames/hoon:<[1.131 7].[1.134 9]>
/sys/vane/ames/hoon:<[1.133 16].[1.133 60]>
/sys/vane/ames/hoon:<[1.615 5].[1.615 43]>
/sys/vane/ames/hoon:<[1.621 5].[1.641 17]>
/sys/vane/ames/hoon:<[1.622 5].[1.641 17]>
/sys/vane/ames/hoon:<[1.624 5].[1.641 17]>
/sys/vane/ames/hoon:<[1.626 5].[1.641 17]>
/sys/vane/ames/hoon:<[1.630 5].[1.641 17]>
/sys/vane/ames/hoon:<[1.724 5].[1.776 62]>
/sys/vane/ames/hoon:<[1.725 5].[1.776 62]>
/sys/vane/ames/hoon:<[1.731 5].[1.776 62]>
/sys/vane/ames/hoon:<[1.735 5].[1.776 62]>
/sys/vane/ames/hoon:<[1.740 5].[1.776 62]>
/sys/vane/ames/hoon:<[1.741 5].[1.776 62]>
[%ames-crash-on-packet-from ~bidbel]
/sys/vane/ames/hoon:<[1.742 5].[1.776 62]>
/sys/vane/ames/hoon:<[1.743 7].[1.743 76]>
/sys/vane/ames/hoon:<[458 3].[469 63]>
/sys/vane/ames/hoon:<[459 3].[469 63]>
/sys/vane/ames/hoon:<[460 5].[460 44]>
[%ames-sndr-tick 0b1]
/sys/vane/ames/hoon:<[460 42].[460 44]>
hear: bail: %exit

[%poke %hear]
bar-stack=[i=[i=//ames t=~] t=~]
call: failed
/sys/vane/ames/hoon:<[1.121 3].[1.153 20]>
/sys/vane/ames/hoon:<[1.123 3].[1.153 20]>
/sys/vane/ames/hoon:<[1.124 3].[1.153 20]>
/sys/vane/ames/hoon:<[1.126 3].[1.153 20]>
/sys/vane/ames/hoon:<[1.127 5].[1.151 7]>
/sys/vane/ames/hoon:<[1.130 5].[1.151 7]>
/sys/vane/ames/hoon:<[1.136 5].[1.151 7]>
/sys/vane/ames/hoon:<[1.138 14].[1.138 53]>
/sys/vane/ames/hoon:<[1.615 5].[1.615 43]>
/sys/vane/ames/hoon:<[1.621 5].[1.641 17]>
/sys/vane/ames/hoon:<[1.622 5].[1.641 17]>
/sys/vane/ames/hoon:<[1.624 5].[1.641 17]>
/sys/vane/ames/hoon:<[1.626 5].[1.641 17]>
/sys/vane/ames/hoon:<[1.630 5].[1.641 17]>
/sys/vane/ames/hoon:<[1.724 5].[1.776 62]>
/sys/vane/ames/hoon:<[1.725 5].[1.776 62]>
/sys/vane/ames/hoon:<[1.731 5].[1.776 62]>
/sys/vane/ames/hoon:<[1.735 5].[1.776 62]>
/sys/vane/ames/hoon:<[1.740 5].[1.776 62]>
/sys/vane/ames/hoon:<[1.741 5].[1.776 62]>
[%ames-crash-on-packet-from ~bidbel]
/sys/vane/ames/hoon:<[1.742 5].[1.776 62]>
/sys/vane/ames/hoon:<[1.743 7].[1.743 76]>
/sys/vane/ames/hoon:<[458 3].[469 63]>
/sys/vane/ames/hoon:<[459 3].[469 63]>
/sys/vane/ames/hoon:<[460 5].[460 44]>
[%ames-sndr-tick 0b1]
/sys/vane/ames/hoon:<[460 42].[460 44]>
crud: bail: %exit
ames: packet failed (4100 total)
```

**Comment by sethfork** [comment]
I noticed a commonality between 3 of the ships I've been getting these ames issues from: `~bidbel`, `~sidlup-havwen` and `~dibref-labter` all appear to have the same kids desk according to network.urbit.org:
`0v1r.0fp5a.98elp.votc5.3ne9d.aei0r.0k1pm.dr69j.iukr2.067p8.2ucn6`

Not sure if that's meaningful, but thought I'd add the observation regardless, in case its useful to someone more knowledgeable  / closer to this part of the stack.

**Comment by jamesacklin** [comment]
We will resolve this as part of #1305. Keeping it around for reproduction/testing.

**Comment by urcades** [comment]
Wonder if this can be closed since #1305 has been completed?
